### PR TITLE
Added i18n support, migrated strings from HTML/TS files to locale en

### LINKED ISSUE #58: Add internationalization
## Expected Behavior

Enabling contributors to easily add support for different languages. Creation of translating service with support of .json formatted language files.

## Actual Behavior

Hardcoded labels, english only.

### PR DESCRIPTION
**Pull request by Darkoon**
Fixes #58 

## Proposed Changes

  - _Added i18n to angular app_
  - _Created `en.json` locale to store current language strings_
  - _Removed strings from ts/html code using translation filter_
  - _Bonus: indentation in imports, added strict 4 spaces rule_

### CODE REVIEW

File: frontend/src/app/components/nav-bar/nav-bar.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
         </ng-container>
         <ng-template #app_brand >
             <img class="app-navbar-brand__logo" src="assets/icon/medtagger_logo.svg" alt="MedTagger logo" />
-            <span class="app-navbar-brand__text">MedTagger</span>
+            <span class="app-navbar-brand__text">{{ 'APP_NAME' | translate }}</span>
```
Comment (kolszewska) [inline frontend/src/app/components/nav-bar/nav-bar.component.html]: I don't think that app name would ever be translated...

Comment (Darkoon) [reply frontend/src/app/components/nav-bar/nav-bar.component.html]: Just a placeholder to bind all occurrences of 'MedTagger' phrase.

File: frontend/src/assets/i18n/en.json [hunk 2]
```diff
@@ -0,0 +1,219 @@
+{
+    "APP_NAME": "MedTagger",
+    "HOME": {
+        "DROPDOWN": {
+            "SETTINGS": "Settings",
+            "LOGOUT": "Log out"
+        },
+        "HEADER": {
+            "GREETING": "Hello",
+            "DESCRIPTION": "MedTagger Home Page"
```
Comment (kolszewska) [inline frontend/src/assets/i18n/en.json]: If we woul leave the APP_NAME, could we use it here? Something along the lines for
`"DESCRIPTION": "APP_NAME +  Home Page"`

Comment (Darkoon) [reply frontend/src/assets/i18n/en.json]: Nope, it should be concatenated in the source. As it is a description, I think its better to have it as one resource string (we can even drop MedTagger phrase here).

**Pull request merged**